english/animals/cats/cats.md

```diff
@@ -0,0 +1,4 @@
+The 'cat' is a small pet animal. It has four short legs and a beautiful furry tail. Its body is covered with soft and silky hairs. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.